config/v1/types_authentication.go

```diff
@@ -20,13 +19,27 @@ type Authentication struct {
 }
 
 type AuthenticationSpec struct {
-	// webhook token auth config (ttl)
-	// external token address
-	// serviceAccountOAuthGrantMethod or remove/disallow it as an option
+	// oauthMetadata contains the discovery endpoint data for OAuth 2.0
+	// Authorization Server Metadata for an external OAuth server.
+	// See IETF Draft:
+	// https://tools.ietf.org/html/draft-ietf-oauth-discovery-04#section-2
```

Comment:
is this just a fancy way of saying, "this is what you serve at .wellknown"?  if so, say that too.

Reply:
This is served from `oc get --raw /asdfklajskdfjasd`